gini/gini-vision-lib-android
# Feature tip popover

Added the pop over that should hint the user to the upload functionality. The popover is shown once and then never again, therefore simple shared preferences have been used.

### How to test
open the camera screen and check if the view is there, close the view. Then kill the app and open it again, nothing should be shown.

### Merging
~do not merge yet since the close asset is missing so far.~ none

### Ticket 
#89
